src/Core/tests/DeviceTests/Handlers/HandlerTestBase.cs

```diff
@@ -21,10 +21,10 @@ public HandlerTestBase(HandlerTestFixture fixture)
 		public Task<T> InvokeOnMainThreadAsync<T>(Func<T> func) =>
 			MainThread.InvokeOnMainThreadAsync(func);
 
-		public Task InvokeOnMainThreadAsync(Action action) =>
+		protected Task InvokeOnMainThreadAsync(Action action) =>
```

Comment:
Any reason for this? Doesn't matter any way since this is only going to be used by derived types anyways

Reply:
Yea, you have to do this otherwise the XUnit Roslyn Analyzers get mad. 

They throw errors like "Please add FACT to all public methods"